backend/controllers/AuthController.ts

```diff
@@ -1,16 +1,16 @@
-import { Response, Request, NextFunction } from 'express';
-import passport from '../libs/passport';
-import mapUserProps from '../mappers/mapUserProps';
-import UserModel, { IUser } from '../models/UserModel';
-import tokenGenerator from '../utils/tokenGenerator';
-import { sendForgotPasswordMail } from '../libs/mailtrap';
-import { v4 as uuidv4 } from 'uuid';
+import { Response, Request, NextFunction } from "express";
+import passport from "../libs/passport";
+import mapUserProps from "../mappers/mapUserProps";
+import UserModel, { IUser } from "../models/UserModel";
+import tokenGenerator from "../utils/tokenGenerator";
+import { sendForgotPasswordMail } from "../libs/mailtrap";
+import { v4 as uuidv4 } from "uuid";
```

Comment:
you should pull and update - because I added eslint and .prettier to the backend - we use single quotes